src/test/univalue_tests.cpp

```diff
@@ -240,11 +240,11 @@ BOOST_AUTO_TEST_CASE(univalue_object)
 
     obj.clear();
     BOOST_CHECK(obj.empty());
-    BOOST_CHECK_EQUAL(obj.count(), 0);
+    BOOST_CHECK_EQUAL(obj.size(), 0);
 }
 
 static const char *json1 =
-"[1.1,{\"key1\":\"str\",\"key2\":800,\"key3\":{\"name\":\"martian\"}}]";
+"[1.10000000,{\"key1\":\"str\",\"key2\":800,\"key3\":{\"name\":\"martian\"}}]";
```

Comment:
To keep the JSON/RPC API backward compatible, all reals represented in JSON have a fixed precision to 8 digits after point.